spec/controllers/idv/in_person/ready_to_verify_controller_spec.rb

```diff
@@ -44,6 +50,43 @@
 
           expect(@analytics).to have_logged_event('IdV: in person ready to verify visited')
         end
+
+        context 'with in_person_proofing_enforce_tmx disabled and pending fraud review' do
+          let!(:profile) { create(:profile, fraud_review_pending_at: 1.day.ago, user: user) }
+          let!(:enrollment) { create(:in_person_enrollment, :passed, user: user, profile: profile) }
+          it 'renders show template' do
+            response
+
+            expect(response).to render_template :show
+          end
+        end
+
+        context 'in_person_proofing_enforce_tmx enabled, pending fraud review, enrollment passed' do
+          let(:in_person_proofing_enforce_tmx) { true }
+          let!(:profile) { create(:profile, fraud_review_pending_at: 1.day.ago, user: user) }
+          let!(:enrollment) { create(:in_person_enrollment, :passed, user: user, profile: profile) }
+
+          it 'redirects to please call' do
+            response
+
+            expect(response).to redirect_to idv_please_call_url
+          end
+        end
+
+        context 'in_person_proofing_enforce_tmx enabled, pending fraud review,
+          enrollment not passed' do
```

Comment:
I think we might actually not want to show the screen in this case but maybe @daviddsilvanava or @eryn-sobing can confirm. The reason I think this is that we currently send the "please call" email to users who get a passed status but not to the users who get a failed status. [Ticket where we implemented that behavior. ](https://cm-jira.usa.gov/browse/LG-12091)

Reply:
Aww shoot, good catch @svalexander. I'm going back to the ticket and finding it's way too vaguely worded. (I wrote it. This is my fault.) This is a tricky gotcha.

I think I would prefer to take discussion of this offline to Slack, but I agree with tagging in David and Eryn and chatting about this tomorrow. It's a question of whether we're consistent with the email we send (I think we should be), or consistent with the behavior in other parts of the app (which I also think we should be, but the two are contradictory here).

Reply:
So @svalexander I realized I messed up the description text for this spec. It's actually working as intended. The 'establishing' enrollment does not redirect to please call, but does render the ready to verify text which is indeed what we want to happen.

Reply:
oh ok so i interpreted "not passed" as "failed" but it's actually "establishing". Can we update the context to `enrollment status equals establishing` or something like that to make the spec super clear?

Reply:
actually maybe that last statement is not needed? looking at the spec in the please_call_controller_spec on line 78 I think that would satisfy my previous suggestion?

Reply:
ok that makes sense